packages/forgae-cli/forgae-node/node.js

```diff
@@ -204,7 +204,7 @@ async function run (option) {
                 // if node is started and error message is another,
                 // we should stop docker
                 
-                await spawn('docker-compose', ['down', '-v'], {});
+                await stopNodeAndCompiler();
```

Comment:
what will happen if the compiler is not up and you try to stop it?

Reply:
command will stop others docker images successfully, it will not throw an exception